OneCommunityGlobal/HGNRest
# Jiyuan added scheduled-on date to scheduled blue square reason

# Description
![Screenshot 2023-11-29 153713](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92963742/37306e7b-3789-44df-8aba-9eff76cba0e4)
![Screenshot 2023-11-29 153739](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92963742/dc81fac0-c8b9-48bd-b171-1c90ad644498)


## Related PRS (if any):
This frontend PR is related to the [#1641](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1641) frontend PR.

## Main changes explained:
- Update file UserProfile.js for adding the `createdDate`

## How to test:
1. check into current branch
2. `npm install`
3. `npm run build`
4. `npm start`
5. follow the instruction of the frontend PR
